src/averageAges.js

```diff
@@ -37,7 +49,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const wasMother = (woman) => (
+    withChildren
+      ? people.some(child => child.mother === woman.name)
+      : true
+  );
```

Comment:
if you have to return true in ternary operator you should use `||` or `&&`. I pointed this out with ternary because I though you will use it sooner (instead of woman.sex check). This solution is good, just change it to `||` or `&&`.
Ex:
`!withChildren || people.some(child => child.mother === woman.name)`